client/i18n/locales/english/intro.json

```diff
@@ -3033,6 +3033,13 @@
       "bnvw": { "title": "242", "intro": [] },
       "xkhk": { "title": "243", "intro": [] },
       "yaxm": { "title": "245", "intro": [] },
+      "workshop-recipe-ingredient-converter": {
```

Comment:
The line `"wyx1": { "title": "247", "intro": [] },` can be removed from `intro.json`.
I am not able to comment at the exact line.